monomer.go

```diff
@@ -50,18 +49,17 @@ type Header struct {
 	Height     int64       `json:"height"`
 	Time       uint64      `json:"time"`
 	ParentHash common.Hash `json:"parentHash"`
-	// state after txs from the *previous* block
-	AppHash  []byte      `json:"app_hash"`
-	GasLimit uint64      `json:"gasLimit"`
-	Hash     common.Hash `json:"hash"`
+	StateRoot  common.Hash `json:"stateRoot"`
```

Comment:
Note to reviewers - the `AppHash` field on the monomer header was repurposed to directly store the ethereum state root